publish.html

```diff
@@ -23,106 +24,80 @@ <h2>Contents</h2>
 
 <div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
+    <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
-
-<img alt="../images/publishDropdownWeb.jpg" class="align-center" src="images/publishDropdownWeb.jpg" style="width: 82%;" />
-
 <div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
+    <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-</li>
-
-
-<li><p class="first">Each script will open in a custom window.</p>
-
-<p>Various parameters can be adjusted, and these values will be used by the script.</p>
-
-<p>Edit the name of the figure or movie if desired.</p>
-
-<p>Select the format of the figure from JPEG, PNG or TIFF.</p>
-
-<p>Some of the script windows differ cosmetically between OMERO.insight and OMERO.web in terms of parameters available for setting and previews of the figures, but both OMERO.insight and OMERO.web use the same scripts, so the eventual output will be the same.</p>
-
-<p>In OMERO.insight click on the <span class="bold">Script</span> button to download or view the script code.</p>
-
-<p>In OMERO.web click on the <span class="bold">View Script</span> link to view the script code.</p>
-
-<p>Use the slider to adjust the size of the previews in OMERO.web script windows.</p>
-
-<p>Click the <span class="bold">Create</span>, <span class="bold">Create Figure</span> or <span class="bold">Make Movie</span> button to run the script.</p>
-
-<p>Figures created by scripts are attached to the Images or the Dataset used in the figure.</p>
-
-<p>Once created, figures can be viewed or downloaded from the activities menu.</p>
-
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
-
-<img alt="../images/publishAll.jpg" class="align-center" src="images/publishAll.jpg" style="width: 100%;" />
-
-</li>
+<p>
+    If you do not have access to OMERO.figure, you can still use the script
+    functionality in OMERO.insight and OMERO.web to create basic figures, see
+    the <a href="#scripts">Using scripts to create figures</a> section below.
+</p>
 
 <p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<li><p class="first">Select <span class="bold">Split View Figure...</span> from the drop-down menu to create a figure with a thumbnail for each channel and one for the full image.</p>
-
-<p>Select the source of labelling from the <span class="bold">Image Label</span> drop-down.</p>
+<h2><a class="anchor" id="movies"></a>Creating movies</h2>
 
-<p>Toggle the Scale Bar on and off and set size and color.</p>
+<ol class="step">
 
-<p>Select Z-sections to be included.</p>
+<li><p class="first">Click on the <span class="bold">Publish</span> icon in the toolbar.</p>
```

Comment:
This could be improved a bit e.g:
"Select a time-lapse Image in the OMEO.insight or OMERO.web client and click on the Publish icon in the toolbar."